astropy/modeling/optimizers.py

```diff
@@ -239,3 +241,122 @@ def __call__(self, objfunc, initval, fargs, **kwargs):
                           "Maximum number of iterations reached.",
                           AstropyUserWarning)
         return fitparams, self.fit_info
+
+
+class Minimize(Optimization):
+    """
+    Interface for Scipy.Optimize.minimize.
```

Comment:
If you write
```
`scipy.optimize.minimize`
```
then sphinx will automatically generate a link when it builds the documentation. That's useful because a user might want to look up the parameters he/she can pass.